knotx-it-tests/src/test/resources/content/local/customAndDefaultSymbol.html

```diff
@@ -42,7 +42,7 @@ <h2>
     </div>
   </div>
   <div class="row">
-    <script data-knotx-knots="databridge,handlebars"
+    <script data-knotx-knots="databridge,te"
```

Comment:
Can we use change snippet options:
1. tag name from `<script>` to `<knotx:snippet>`
2. attribute prefix = '', ex. `knots="..."`